### DISCUSSION TITLE
SCP plugin not working: remotior sensus outdated

### DISCUSSION THREAD

**Discussion by ensbenz12** [semiautomaticgit/SemiAutomaticClassificationPlugin#384] [General]
Hello, i got myself QGIS version 3.38 Grenoble for my university studies. Recently i downloaded the SCP plugin, but i can't open it: it tells me that the python library remotior sensus is outdated and needs and update. Where can i do this update?. Searching for a solution i found out to just import an older version of the SCP dock, but even there certain things dont work. So right now im turning in circles.
is someone able to help me with this issue?

thx in advance

**Comment by semiautomaticgit** [comment]
Hello @ensbenz12 , 
please install the plugin following these steps from the user manual https://semiautomaticclassificationmanual.readthedocs.io/en/latest/installation.html  .

**Reply by ensbenz12** [reply]
hello, thx for the advice. for me it worked with the: pip3 install --upgrade remotior-sensus command

**Reply by semiautomaticgit** [reply]
> hello, thx for the advice. for me it worked with the: pip3 install --upgrade remotior-sensus command

You are welcome, I'm glad it worked
